cloudinit/net/bsd.py

```diff
@@ -85,29 +84,70 @@ def _ifconfig_entries(self, settings):
                         "mtu": subnet.get("mtu") or interface.get("mtu"),
                     }
 
+                elif subnet.get("type") == "static6":
+                    if not subnet.get("prefix"):
+                        LOG.debug(
+                            "Skipping IP %s, because there is no prefix",
+                            subnet.get("address"),
+                        )
+                        continue
+                    LOG.debug(
+                        "Configuring dev %s with %s / %s",
+                        device_name,
+                        subnet.get("address"),
+                        subnet.get("prefix"),
+                    )
+
+                    self.interface_configurations_ipv6[device_name] = {
+                        "address": subnet.get("address"),
+                        "prefix": subnet.get("prefix"),
+                        "mtu": subnet.get("mtu") or interface.get("mtu"),
+                    }
+                elif (
+                    subnet.get("type") == "dhcp"
+                    or subnet.get("type") == "dhcp4"
+                ):
+                    self.interface_configurations[device_name] = "DHCP"
+
```

Comment:
I see nothing for dynamic IPv6. Will this come in a new PR?

Reply:
We don't use any dynamic IPv6, so I left it out.

Reply:
well it's still a very good start, thank you.